pages/Documents/ClientSideConfiguration/ServerChatAPI/Methods/retrieve-pre-chat-survey.md

```diff
@@ -45,7 +45,14 @@ Returns an XML or JSON response with the pre-chat survey.
 | visitorId | The current visitor ID. | alphanumeric | This parameter can be used to associate the survey to the current visitor.|
 | skill | Set a skill to the current visitor. | alphanumeric | |
 
-<div class="attn-note">To retrieve the <code>surveyApiId</code>, you need to call the Engagement and Window API. The information to retrieve the <code>domain</code> name can be found <a href="/agent-domain-domain-api.html">here</a>. Make sure to set the service name as <code>acCdnDomain</code>. <br><br>Through the Engagement API, you need to call <code>GET https://{domain}/api/account/{accountId}/configuration/le-campaigns/campaigns/{campaignId}/engagements/{engagementId}/revision/{revisionId}?v=3.0</code> to get the <code>windowId</code>. <br><br>Next, through the Window API, you need to call <code>GET https://{domain}/api/account/{accountId}/configuration/engagement-window/window-confs/{windowId}</code> to get the <code>surveyPreChatId</code>. The <code>surveyPreChatId</code> is the value that needs to be passed through the <code>surveyApiId</code> parameter.</div>
```

Comment:
This is one note, not three, and a use case for allowing HTML for multi-paragraph notes.

If you don’t want to ensure HTML support for notes right now, make this one large paragraph—but please take note to split this up again into several paragraphs once multi-paragraph support is ensured.

(Let me know if you have any questions or concerns.)

Reply:
fixed